jenkins/promotion/tar-promotion.jenkinsfile

```diff
@@ -0,0 +1,60 @@
+lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 1, unit: 'HOURS')
+    }
+    agent none
+    parameters {
+        choice(
+            choices: ['distribution-build-opensearch', 'distribution-build-opensearch-dashboards'],
+            name: 'DISTRIBUTION_JOB_NAME',
+            description: 'What platform is this distribution build for?.'
+        )
+        string(
+            defaultValue: '',
+            name: 'DISTRIBUTION_BUILD_NUMBER',
+            description: 'What is the build id of the above DISTRIBUTION_JOB_NAME that you want to promote? (e.g. 123, 136)',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'INPUT_MANIFEST',
```

Comment:
For follow up, I think we can vastly reduce the number of parameters if we feed in a build manifest file as it has many of these parameters in it, and its a better reference.

Reply:
Yes, as this is a startup for the task, it is a bit complicated in the logic.
Can fine-tune later will add to the issue.
Thanks.